### PR TITLE
using JuMP utilities for MOI in demo app

### PR DESCRIPTION
**Pull request by IssamT**

### CODE REVIEW

File: demos/GeneralizedAssignment_ColGen/run_sgap.jl [hunk 1]
```diff
@@ -1,4 +1,5 @@
 using JuMP
+using MathOptInterface: set
```
Comment (vitornesello) [inline demos/GeneralizedAssignment_ColGen/run_sgap.jl]: I don't think we should do this. We will lose consistency throughout the code. `MOI.` is ver short already

Comment (IssamT) [reply demos/GeneralizedAssignment_ColGen/run_sgap.jl]: The idea is to have JuMP applications written in a friendly language. that's why for example we don t write JuMP.@variable or JuMP.Model

Comment (vitornesello) [reply demos/GeneralizedAssignment_ColGen/run_sgap.jl]: Ok, thats only in the JuMP level

Comment (IssamT) [reply demos/GeneralizedAssignment_ColGen/run_sgap.jl]: Yes exactly and the MOI.set i use are the shortcuts defined in JuMP such as https://github.com/JuliaOpt/JuMP.jl/blob/2ae979eec4aeac1b6dc76d614b79c3c99c3dacc5/src/JuMP.jl#L468

**Pull request merged**